contact_links/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'AF Contact Links',
-    'version': '12.0.0.2.1',
+    'version': '12.0.0.2.2',
```

Comment:
This commit seems to be adding functionality, therefore the version number should be 12.0.0.3.0

Reply:
Ok @nils-vertel

Reply:
Fixed @nils-vertel